registry/Dockerfile

```
@@ -3,7 +3,7 @@
 # Stage1: build from source
 FROM quay.io/cybozu/golang:1.19-jammy AS build
```

Please update the version of golang as well.
```suggestion
FROM quay.io/cybozu/golang:1.20-jammy AS build
```